backend/src/v5/middleware/dataConverter/outputs/auth.js

```diff
@@ -0,0 +1,74 @@
+/**
+ *  Copyright (C) 2021 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+const config = require('../../../utils/config');
+const { events } = require('../../../services/eventsManager/eventsManager.constants');
+const { getURLDomain } = require('../../../utils/helper/strings');
+const { isFromWebBrowser } = require('../../../utils/helper/userAgent');
+const { publish } = require('../../../services/eventsManager/eventsManager');
+const { respond } = require('../../../utils/responder');
+const { templates } = require('../../../utils/responseCodes');
+
+const Auth = {};
+
+Auth.regenerateAuthSession = (req, res) => {
+	req.session.regenerate((err) => {
+		if (err) {
+			respond(req, res, templates.sessionCouldBeRegenerated);
```

Comment:
do you mean `sessionCouldNotBeRegenerated`?

eitherway this isn't a very useful error to the user and if we can't generate a session, there's an internal error, log the error on the `logger` here and respond with the error (which most likely will be translated to `unknownError`)